src/fs/home.go

```diff
@@ -7,12 +7,11 @@ import (
 	"github.com/peterebden/go-deferred-regex"
 )
 
-var home = os.Getenv("HOME")
 var homeRex = deferredregex.DeferredRegex{Re: "(?:^|:)(~(?:[/:]|$))"}
 
 // ExpandHomePath expands all prefixes of ~ without a user specifier to $HOME.
 func ExpandHomePath(path string) string {
-	return ExpandHomePathTo(path, home)
+	return ExpandHomePathTo(path, os.Getenv("HOME"))
```

Comment:
Moved this here otherwise the value would be stale. I faced this issue when writing tests, but it seems also reasonable otherwise.